src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor

```diff
@@ -25,6 +25,24 @@
                 // This takes up too much horizontal space on mobile, so show on a new line on mobile
                 <FluentLabel Typo="Typography.Body" aria-live="polite" aria-label="@Loc[nameof(Dashboard.Resources.ConsoleLogs.LogStatusLabel)]">@PageViewModel.Status</FluentLabel>
             }
+
+            @if (PageViewModel.SelectedResource is not null)
+            {
+                <FluentMenuButton ButtonAppearance="@Appearance.Neutral"
```

Comment:
The menu button should always be visible. If there is no selected resource then disable the menu items.